src/containers/Profiles/Form.tsx

```diff
@@ -41,7 +41,12 @@ const Form = ({ initialValues, formHandler, selectedAvatar, showCancelButton = t
   ];
 
   const validationSchema: SchemaOf<{ name: string }> = object().shape({
-    name: string().required(t('profile.validation.name.required')).min(2, t('profile.validation.name.too_short')),
+    name: string()
```

Comment:
recheck the validation.. seems like it has missing translation, as well as misleading validation message. 

<img width="694" alt="image" src="https://github.com/jwplayer/ott-web-app/assets/37805865/b5857212-1d81-49e7-abb1-d983bca856cb">
<img width="694" alt="image" src="https://github.com/jwplayer/ott-web-app/assets/37805865/182637d1-34b3-4c0d-b81b-e429271b07db">